kfs-core/src/main/java/edu/arizona/kfs/fp/document/validation/impl/DisbursementVoucherNonEmployeeTravelValidation.java

```diff
@@ -0,0 +1,214 @@
+
+	private boolean isTravelNonEmplPaymentReason(DisbursementVoucherDocument disbursementVoucherDocument) {
+		ParameterEvaluator travelNonEmplPaymentReasonEvaluator = SpringContext.getBean(ParameterEvaluatorService.class).getParameterEvaluator(DisbursementVoucherDocument.class, DisbursementVoucherConstants.NONEMPLOYEE_TRAVEL_PAY_REASONS_PARM_NM, disbursementVoucherDocument.getDvPayeeDetail().getDisbVchrPaymentReasonCode());
+		return travelNonEmplPaymentReasonEvaluator.evaluationSucceeds();
+	}
+
+	private boolean validatePerDiemSection(DisbursementVoucherDocument document, MessageMap errors) {
+		boolean perDiemSectionComplete = true;
+
+		// Checks to see if any per diem fields are filled in
+		boolean perDiemUsed = StringUtils.isNotBlank(document.getDvNonEmployeeTravel().getDisbVchrPerdiemCategoryName()) || ObjectUtils.isNotNull(document.getDvNonEmployeeTravel().getDisbVchrPerdiemRate()) || ObjectUtils.isNotNull(document.getDvNonEmployeeTravel().getDisbVchrPerdiemActualAmount());
+
+		// If any per diem fields contain data, validates that all required per
+		// diem fields are filled in
+		if (perDiemUsed) {
+			if (StringUtils.isBlank(document.getDvNonEmployeeTravel().getDisbVchrPerdiemCategoryName())) {
+				errors.putError(KFSPropertyConstants.DISB_VCHR_PERDIEM_CATEGORY_NAME, KFSKeyConstants.ERROR_DV_PER_DIEM_CATEGORY);
+				perDiemSectionComplete = false;
+			}
+			if (ObjectUtils.isNull(document.getDvNonEmployeeTravel().getDisbVchrPerdiemRate())) {
+				errors.putError(KFSPropertyConstants.DISB_VCHR_PERDIEM_RATE, KFSKeyConstants.ERROR_DV_PER_DIEM_RATE);
+				perDiemSectionComplete = false;
+			}
+			if (ObjectUtils.isNull(document.getDvNonEmployeeTravel().getDisbVchrPerdiemActualAmount())) {
+				errors.putError(KFSPropertyConstants.DISB_VCHR_PERDIEM_ACTUAL_AMOUNT, KFSKeyConstants.ERROR_DV_PER_DIEM_ACTUAL_AMT);
+				perDiemSectionComplete = false;
+			}
+		}
+		perDiemSectionComplete = perDiemSectionComplete && perDiemUsed;
+		return perDiemSectionComplete;
+	}
+
+	private boolean validatePersonalVehicleSection(DisbursementVoucherDocument document, MessageMap errors) {
+		boolean personalVehicleSectionComplete = true;
+
+		// Checks to see if any per diem fields are filled in
+		boolean personalVehilcleUsed = ObjectUtils.isNotNull(document.getDvNonEmployeeTravel().getDisbVchrAutoFromCityName())
+				|| ObjectUtils.isNotNull(document.getDvNonEmployeeTravel().getDisbVchrAutoFromStateCode())
+				|| ObjectUtils.isNotNull(document.getDvNonEmployeeTravel().getDisbVchrAutoToCityName())
+				|| ObjectUtils.isNotNull(document.getDvNonEmployeeTravel().getDisbVchrAutoToStateCode())
+				|| ObjectUtils.isNotNull(document.getDvNonEmployeeTravel().getDvPersonalCarMileageAmount())
+				|| ObjectUtils.isNotNull(document.getDvNonEmployeeTravel().getDisbVchrMileageCalculatedAmt())
+				|| ObjectUtils.isNotNull(document.getDvNonEmployeeTravel().getDisbVchrPersonalCarAmount());
+
+		// If any per diem fields contain data, validates that all required per
```

Comment:
Correct copy-paste comment; still references "per diem" in "personal vehicle" section